_1327/documents/markdown_internal_links.py

```diff
@@ -0,0 +1,54 @@
+from django.core.urlresolvers import reverse
+import markdown
+from markdown.inlinepatterns import LinkPattern
+
+from _1327.information_pages.models import InformationDocument
+from _1327.minutes.models import MinutesDocument
+from _1327.polls.models import Poll
+
+
+class InternalLinkPattern (LinkPattern):
+
+	def handleMatch(self, m):
+		el = markdown.util.etree.Element("a")
+		el.set('href', self.url(m.group('id')))
+		el.text = markdown.util.AtomicString(m.group('title'))
+		return el
+
+	def url(id):
+		raise NotImplementedError
+
+
+class InternalLinkMinutesPattern (InternalLinkPattern):
```

Comment:
These should be in the respective Django app, not in documents. And wouldn't it be nice to implement the interface in the actual model?